gen/flights/flightplan.py

```diff
@@ -411,6 +411,9 @@ class PatrollingFlightPlan(FlightPlan):
     #: Maximum time to remain on station.
     patrol_duration: timedelta
 
+    #: Racetrack speed TAS.
+    patrol_speed: Speed
```

Comment:
This breaks save compat. In develop-4.x you'll need to populate this when it's missing using `__setstate__` or work around its absence wherever it's used. In either case, remember to add the `@has_save_compat_for(4)` so we don't lose track of it.